doc/py_cell.rst

```diff
@@ -0,0 +1,153 @@
+.. _pycell:
+
+Cells
+=====================
+
+Cell Identifiers and Indexes
+----------------------------
+The types defined below are used as identifiers for cells and members of cell-local collections.
+
+.. module:: arbor
+
+.. class:: cell_member
+
+    .. function:: cell_member(gid, index)
+
+        Construct a cell member with parameters :attr:`gid` and :attr:`index` for global identification of a cell-local item.
```

Comment:
'Construct a cell member' doesn't really make sense; should be e.g. 'Construct a `cell_member` object'.